features.json

```diff
@@ -95,13 +95,14 @@
 				"bulkMemory": "75",
 				"exceptions": "95",
 				"extendedConst": ["flag", "Requires flag `--js-flags=--experimental-wasm-extended-const`"],
+				"memory64": ["flag", "Requires flag `--js-flags=--experimental-wasm-memory64`"],
```

Comment:
Out of curiousity, does this apply to Node.js too? Usually both share the same V8 flags (if Node.js updated its V8 already).

Reply:
Yeah, I assume it does (as you said, they are usually the same) although I haven't tested it.